core/src/test/scala/unit/kafka/coordinator/transaction/TransactionStateManagerTest.scala

```diff
@@ -875,18 +875,35 @@ class TransactionStateManagerTest {
     val startOffset = 0L
     val records = MemoryRecords.withRecords(startOffset, CompressionType.NONE, txnRecords.toArray: _*)
 
+    val transactionsWithPendingMarkers = new ConcurrentHashMap[String, PendingCompleteTxn]
+    def sendMarkers(coordinatorEpoch: Int,
+                    txnResult: TransactionResult,
+                    txnMetadata: TransactionMetadata,
+                    newMetadata: TxnTransitMetadata): Unit = {
+      val transactionalId = txnMetadata.transactionalId
+      val pendingCompleteTxn = PendingCompleteTxn(
+        transactionalId,
+        coordinatorEpoch,
+        txnMetadata,
+        newMetadata)
+
+      transactionsWithPendingMarkers.put(transactionalId, pendingCompleteTxn)
+    }
+
     prepareTxnLog(topicPartition, 0, records)
 
     // immigrate partition at epoch 0
-    transactionManager.loadTransactionsForTxnTopicPartition(partitionId, coordinatorEpoch = 0, (_, _, _, _) => ())
+    transactionManager.loadTransactionsForTxnTopicPartition(partitionId, coordinatorEpoch = 0, sendMarkers, false)
```

Comment:
Since this is a Scala test class can you use named arguments to improve readability similar to the argument `coordinatorEpoch = 0`?